.github/workflows/build-release.yml

```diff
@@ -54,6 +54,21 @@ jobs:
           ./build/flow-php.phar --version
           ./build/flow-php.phar examples/topics/transformations/array_expand.php
 
+      - name: "Import GPG Key"
+        uses: crazy-max/ghaction-import-gpg@72b6676b71ab476b77e676928516f6982eef7a41
```

Comment:
Why use specific commit for that action? Is there something broken in later versions?

Reply:
This action is touching something very sensitive, I think it's better to lock it to a specific commit which minimizes the risk of someone pushing something nasty into that repo